### ISSUE TITLE
A bad experience about EUD Editor

### ISSUE THREAD

**Issue by boermyth** [Buizz/EUDEditor#2]
The EUD Editor is great and thank the maker very much, but I think if it doesn't need to link to it's server otherwise you can't use it's Insert function what is a bad experience for me or others, it will be more better. 

Of course, my computer is on the Internet but the euddraft's server is often unreachable in our region when I use the version which is newer than euddraft 0.8.2.4, and I can't use euddraft 0.8.2.4 because of some new functions (It means the versions after 0.8.2.4 must be linked).Now I use euddraft 0.8.4.7.(More newer version such as 0.8.6.0 will make my map has some bugs)

In simple terms, most of time, we can't use EUD Editor in our region because of it's networking mechanism, even though I just want to have a test in map. This makes us very depressed. I don't know why it must link to it's server, I hope it can be used even though without any internet and has more humanization designs which considering all kinds of potential problems and different environments.

However, the tool is great and thank the maker again.

**Comment by armoha** [comment]
Hi, I'm current maintainer of `euddraft`, since version `0.8.3.0`.

The euddraft autoupdate uses github server, so autoupdate does *not* work if github is blocked in your internet service.

I fixed a bug related to EUD Editor's `SetDatfile` and `AddDatfile` action, and released `0.8.6.1` yesterday. Could you try `euddraft 0.8.6.1` and if bug consists in the map, reporting bug to me would be appreciated.

- euddraft repo link: https://github.com/armoha/euddraft/tree/master/latest

AFAIK `EUD Editor 2` requires internet connection only when it tries to check version for autoupdate. I'm using `EUD Editor 2 0.17.8.2`, and when I disconnected internet and tried `NewFunc`, `Load Func` and inserting `Func`, it looks like they working so I couldn't reproduce your issue. Could you elaborate it more?

Indeed `EUD Editor 2` is not ergonomic in some aspects. @Buizz is working on `EUD Editor 3` and there will be improvements for more user-friendly tool.

**Comment by boermyth** [comment]
Thank you for your reply. And I cann't download the EUD Editor 3.0 in https://cafe.naver.com/edac. And we cann't register there. It seems there is no EUD Editor 3.0 here.

By the way, the EUD Editor is a very cool map tool which makes rpg more wonderful, if the new version can support Simplified Chinese (GB2312), it will be a great gift for Chinese map-makers, there are many EUD Editor's users in China. However, I believe it will become better and better.

**Comment by Buizz** [comment]
https://github.com/Buizz/EUD-Editor-3
This is the EUD Editor3 Github. Here you can build and run the latest version of EUD Editor3.


# Chinese translation
https://github.com/Buizz/EUD-Editor-3/tree/master/EUD%20Editor%203/Language
If the user translates the language file in the request, it can be supported immediately.

Thank you for using my tool.
